### PR TITLE
Add com.kalbareader.Kalba

### PR DESCRIPTION
**Pull request by BrewingWeasel**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
Kalba is a sentence mining tool that helps users read in a foreign language and automatically create flashcards from the content they consume.

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed. 

Yes, I control http://kalbareader.com (evidence: it redirects to the github repo)

- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. 
I am the creator (https://github.com/brewingweasel/kalba)

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: com.kalbareader.Kalba.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - --share=network
+
+modules:
+  - name: kalba
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/BrewingWeasel/Kalba/releases/download/kalba-v0.3.2/Kalba_0.3.2_amd64.deb
+        sha256: 23d0186a436f6e30b07024fc8f0e4359d8bc35e35c854941745a057688e7bc4a # This is required if you are using a remote source
+        only-arches: [x86_64] #This source is only used on x86_64 Computers
```
Comment (bbhtt) [inline com.kalbareader.Kalba.yml:23]:
```suggestion
        sha256: 23d0186a436f6e30b07024fc8f0e4359d8bc35e35c854941745a057688e7bc4a # This is required if you are using a remote source
        only-arches: [x86_64] #This source is only used on x86_64 Computers
        dest-filename: Kalba.deb
```

File: com.kalbareader.Kalba.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        # This path points to the binary file which was created in the .deb bundle.
+        # Tauri also creates a folder which corresponds to the content of the unpacked .deb.
+    build-commands:
+      - ar -x *.deb
+      - tar -xf data.tar.gz
```
Comment (bbhtt) [inline com.kalbareader.Kalba.yml:28]:
```suggestion
      - bsdtar -Oxf Kalba.deb 'data.tar.gz' | bsdtar -xf -
```

File: com.kalbareader.Kalba.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      - install -Dm755 usr/bin/kalba /app/bin/kalba
+      - sed -i 's/Categories=/Categories=Languages;Education/' usr/share/applications/kalba.desktop
+      - sed -i 's/Icon=kalba/Icon=com.kalbareader.Kalba/' usr/share/applications/kalba.desktop
```
Comment (bbhtt) [inline com.kalbareader.Kalba.yml:31]: Use desktop-file-edit here.

File: com.kalbareader.Kalba.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      - install -Dm644 usr/share/applications/kalba.desktop /app/share/applications/com.kalbareader.Kalba.desktop
+      - install -Dm644 usr/share/icons/hicolor/128x128/apps/kalba.png /app/share/icons/hicolor/128x128/apps/com.kalbareader.Kalba.png
+      - install -Dm644 usr/share/icons/hicolor/32x32/apps/kalba.png /app/share/icons/hicolor/32x32/apps/com.kalbareader.Kalba.png
+      - install -Dm644 usr/share/icons/hicolor/256x256@2/apps/kalba.png /app/share/icons/hicolor/256x256@2/apps/com.kalbareader.Kalba.png
+      - install -Dm644 usr/share/metainfo/com.kalbareader.Kalba.metainfo.xml com.kalbareader.Kalba.metainfo.xml
```
Comment (bbhtt) [inline com.kalbareader.Kalba.yml:36]:
```suggestion
      - install -Dm644 usr/share/metainfo/com.kalbareader.Kalba.metainfo.xml /app/share/metainfo/com.kalbareader.Kalba.metainfo.xml
```

File: flathub.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
+}
```
Comment (bbhtt) [inline flathub.json:3]: Why not build from source?

Comment (BrewingWeasel) [reply flathub.json:3]: I'd like to eventually, but for now I'm trying to just follow the path of least resistance and do what the tauri docs and other apps I've seen do

**Comment by BrewingWeasel** [comment]
bot, build com.kalbareader.Kalba

**Comment by bbhtt** [comment]
bot, build com.kalbareader.Kalba

**Pull request closed without merging**